test/CMakeLists.txt

```diff
@@ -163,15 +163,40 @@ add_test(
 # tests are compiled without compiler extensions to ensure the stlab headers
 # are not dependent upon any such extension.
 #
-set_target_properties(
-  stlab.test.channel
-  stlab.test.future
-  stlab.test.serial_queue
-  stlab.test.cow
-  stlab.test.task
-  stlab.test.tuple
-  stlab.test.traits
-  PROPERTIES CXX_EXTENSIONS OFF )
+
+if (EMSCRIPTEN)
```

Comment:
@camio can you please suggest a way to make this less repetitive? I wasn't aware of an idiomatic solution.